pages/learn/manage/device-statuses.md

```diff
@@ -11,36 +11,69 @@ Device statuses are displayed on the Devices page and the Device Summary page. A
+| **Operational**                              | The device is functioning normally and communicating with the cloud within the expected interval. In the standard configuration, this means both [cloudlink][cloudlink] and the device heartbeat API are working. If [cloudlink is disabled][bandwidth-reduction], the device will still appear `Operational` if the heartbeat API is functional.   |
+| **Reduced Functionality**<sup>1</sup>        | The device is connected to the backend but is experiencing limited functionality due to either **missing recent Heartbeat API communications** or **lack of connection to cloudlink**.                                                                                       |
+| **Disconnected**                                  | The device is not connected to cloudlink and has not communicated via the Heartbeat API for a duration exceeding the [API polling interval][poll-interval]. This status indicates that the device is completely offline and unable to send or receive any data from the cloud.                                                                                                                                                           |
+| **Configuring**                              | The device is applying OS configuration. For example, an OS upgrade.                                                                                                                                         |
+| **Updating**                                 | The device is updating to a new App release. In future versions this will also reflect OS updates.                                                                                                                                        |
 | **Post Provisioning**                        | The device has been [provisioned][device-provisioning] but has not yet come online.                                                                                              |
 | **Inactive**                                 | The device has been [deactivated][deactivated] or has been [preregistered][preregistered] but has not yet connected to the {{ $names.cloud.lower }} API.                         |
 | **Frozen**                                   | The device has been frozen because it's outside the organization's allowance, or is in a paid [fleet type][fleet type] on a free tier organization.                              |
 
+
 ## Device Connectivty states
 
+
+
+The balena device uses two mechanisms to maintain communication with balenaCloud, namely a periodic HTTP API heartbeat and a VPN based tunnel called cloudLink. The state of both of these links can be seen in the device listing.
+
 ![Device connectivity indicators](/img/common/main_dashboard/device_status.webp)
```

Comment:
Is it overkill to explain why a user would see '-' instead of a status? I know we've added a note of 'data for these states will only get populated for devices with supervisor 16.7.0 or later' but should we have a catch all as to why this shows? e.g. there's no data to populate this section

Reply:
That feels a bit overkill